applications/MeshingApplication/CMakeLists.txt

```diff
@@ -59,6 +59,40 @@ endif(${KRATOS_BUILD_TESTING} MATCHES ON)
 add_library(KratosMeshingCore SHARED ${KRATOS_MESHING_APPLICATION_CORE} ${KRATOS_MESHING_APPLICATION_CORE_MMG} ${KRATOS_MESHING_APPLICATION_TESTING_SOURCES})
 
 if(${INCLUDE_MMG} MATCHES ON)
+    # Define includes and libraries with just one include
+    IF (MMG_ROOT)
+        SET(MMG_INCLUDE_DIR ${MMG_ROOT}/include/mmg/)
+        SET(MMG2D_INCLUDE_DIR ${MMG_ROOT}/include/mmg/mmg2d/)
+        SET(MMG3D_INCLUDE_DIR ${MMG_ROOT}/include/mmg/mmg3d/)
+        SET(MMGS_INCLUDE_DIR ${MMG_ROOT}/include/mmg/mmgs/)
+        IF(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+            IF(EXISTS "${MMG_ROOT}/lib/libmmg.dll") # Dynamic library
+                SET(MMG_LIBRARY ${MMG_ROOT}/lib/libmmg.dll)
+                SET(MMG2D_LIBRARY ${MMG_ROOT}/lib/libmmg2d.dll)
+                SET(MMG3D_LIBRARY ${MMG_ROOT}/lib/libmmg3d.dll)
+                SET(MMGS_LIBRARY ${MMG_ROOT}/lib/libmmgs.dll)
+            ELSE(EXISTS "${MMG_ROOT}/lib/libmmg.dll") # Static library
```

Comment:
static and dynamic are the same? Why the `if` then?
Did anyone compile with mmg in win or is this just a guess?

Reply:
They are not the same, is the way Cmake conditionals work